src/h5web/visualizations/heatmap/AxisGrid.tsx

```diff
@@ -3,20 +3,16 @@ import { useThree, useFrame, Dom } from 'react-three-fiber';
 import { scaleLinear } from 'd3-scale';
 import IndexAxis from './IndexAxis';
 import styles from './HeatmapVis.module.css';
-import { Domain } from './store';
+import { useProps } from './hooks';
+import { Domain } from './models';
```

Comment:
Now that the store no longer contains all the computations, I've moved all shared models to `models.ts`.